app/src/main/java/com/googleresearch/capturesync/PhaseAlignController.java

```diff
@@ -92,11 +120,24 @@ private void doPhaseAlignStep() {
                         latestResponse.exposureTimeToShiftNs() * 1e-6f,
                         phaseAligner.getConfig().minExposureNs() * 1e-6f));
 
-        // TODO(samansari): Make this an interface.
-        // context.injectFrame(latestResponse.exposureTimeToShiftNs());
+        final CameraController cameraController = context.getPreview().getCameraController();
```

Comment:
The problem is that a camera controller may be changed while phase alignment is running. I'll modify phase alignment so it stops when such thing happens, and then it will be possible to save camera controllers in a field.